static/sass/main.scss

```diff
@@ -1,51 +1,28 @@
-// import required files
+// import vanilla-brochure-theme
+@import '../../node_modules/vanilla-brochure-theme/scss/theme';
```

Comment:
We include node_modules when compiling scss: https://github.com/canonical-websites/maas.io/blob/master/package.json#L8